__tests__/api.test.js

```diff
@@ -147,32 +146,46 @@ describe("8. GET /api/articles?topic", () => {
   
   });
 
-  
 
-// ------------ Error Handling ------------//
-describe("Error Handling", () => {
-  test("404: show user that URL is invalid", () => {
-    return request(app)
-      .get("/api/BigManTing")
-      .expect(404)
-      .then((res) => {
-        expect(res.body.msg).toBe("Invalid URL");
-      });
-  });
-});
-test('invalid ID datatype provided for getArticleById', () => {
-  return request(app)
-    .get('/api/articles/SELECTA!')
-    .expect(400)
-    .then((res) => {
-      expect(res.body).toEqual({ msg: 'Invalid id type!' });
+  describe('9. GET /api/articles/:article_id/comments', () => {
+    test("200: returns comments for article", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((res) => {
+          expect(res.body).toBeSortedBy("created_at", {
+            descending: true,
+          });
+          res.body.forEach((comment) => {
+            expect(Object.keys(comment)).toEqual([
+              "comment_id",
+              "body",
+              "article_id",
+              "author",
+              "votes",
+              "created_at"
+            ]);
+          });
+        });
+        });
+
+    test("404: returns msg for article with no comments", () => {

```

Comment:
this test description is a little confusing, are we 404ing for an article with no comments or are we 404ing for an article that doesnt exist?
If its no comments I would probably have it 200 if the article exists and doesnt have any comments, and expect it to return an empty array